### DISCUSSION TITLE
New logo concept

### DISCUSSION THREAD

**Discussion by daniel4-scratch** [ScratchAddons/ScratchAddons#3751] [Show and tell]
![Scratch Addons](https://user-images.githubusercontent.com/65277548/140635006-72dfac26-a991-445c-8bbe-021fe08e0867.png)
I made and improved logo concept, does anyone like it?

**Comment by cobaltt7** [comment]
lol nice


________________________________

**Comment by WorldLanguages** [comment]
Looks good
The problem with most suggested logos is that people forget we have very few pixels to play with in the extension bar, typically around 20x20. So things like shadows and rounded corners wouldn't be noticeable or wouldn't look good.

**Reply by daniel4-scratch** [reply]
This logo could be used in typical large areas like on a website

**Comment by Secret-chest** [comment]
I would like it to be a circle

**Comment by Mbrick2** [comment]
YESSSS

**Comment by Secret-chest** [comment]
Can you make the lego not rounded? Also, I believe you copied someone's work (it was posted before)

**Reply by daniel4-scratch** [reply]
I posted here with a different logo, then I posted the new one. I like that one better so I made it the icon. (Check the post’s history)
